src/pages/team/index.tsx

```diff
@@ -159,15 +167,55 @@ const Team = ({team: teamData}: TeamPageProps) => {
               label={true}
             />
           </div>
-          <AtCapacityNotice teamData={teamData} />
+          <AtCapacityNotice
+            isFull={teamData.isFull}
+            billingPortalUrl={subscriptionData.portalUrl}
+            billingScheme={subscriptionData.billingScheme}
+          />
           <h2 className="font-semibold text-xl mt-16">
             Current Team Members <TeamComposition teamData={teamData} />
           </h2>
           <MemberTable members={teamData.members} />
           <BillingSection
-            stripeCustomerId={teamData.stripeCustomerId}
-            slug={teamData.accountSlug}
+            subscriptionData={subscriptionData}
+            loading={subscriptionDataLoading}
           />
+          {!subscriptionDataLoading &&
+            subscriptionData.billingScheme === 'per_unit' && (
```

Comment:
This section displays an additional note about adding seats to your account based IF this account is on legacy pricing.